samples/detect.js

```diff
@@ -23,24 +23,40 @@ const {Transform, pipeline} = require('stream');
 const pump = util.promisify(pipeline);
 
 function detectTextIntent(projectId, sessionId, queries, languageCode) {
+  const PROJECT_ID = projectId;
+  const SESSION_ID = sessionId;
+  const QUERIES = queries;
+  const LANGUAGE_CODE = languageCode;
   // [START dialogflow_detect_intent_text]
+
+  /**
+   * TODO(developer): UPDATE these variables before running the sample.
+   */
+  // const PROJECT_ID = 'PROJECT_ID';
+  // const SESSION_ID = String(Date.now()); // sessionId can be a random number or some type of user identifier (preferably hashed)
```

Comment:
Why not use `Math.floor(Math.random() * 100000)` to show that it's random? Actually, I think I'd prefer a place holder:

const SESSION_ID = 123456;  // Random number or hashed user identifier

Reply:
Sure.